docs/blog/2020-01-08-git-workflows/index.md

```diff
@@ -0,0 +1,288 @@
+---
+title: "Incremental PRs: a new Github workflow for the Cloud team"
+date: 2020-01-30
+author: "Josh Comeau"
+excerpt: "A look at how the Cloud team adopted a new workflow to achieve smaller, non-blocking PRs."
+tags: ["source control", "Github", "workflows"]
+---
+
+import baseGithub from "./base-github.png"
+import initialSvg from "./1-initial.svg"
+import additionalWorkSvg from "./2-additional-work.svg"
+import rebasedSvg from "./3-rebased.svg"
+import mergeSrc from "./merge.png"
+import mergeCommitSvg from "./4-merge-commit.svg"
+import actualGitSvg from "./6-actual-git.svg"
+import parallelUniversesSvg from "./7-parallel-universes.svg"
+import doubleWorkSvg from "./8-double-work.svg"
+import moreIncrementalSvg from "./5-more-incremental.svg"
+
+2020 is shaping up to be a really exciting year for Gatsby, with lots of ambitious projects on the roadmap. On the Cloud team, we've been getting all of our ducks in a row, making sure we're all set to hit the ground running. One of the areas we've been focusing on is our Github workflow.
+
+We were seeing a lot of "stalled" Pull Requests (PRs) - work was being put up for review, but not receiving prompt attention. These PRs tended to be quite large and complex, essentially containing the entirety of work for a given feature or refactor.
+
+Monolithic PRs are difficult to review. In addition to the time investment, they also tend to require a lot of mental energy; to be an effective reviewer, you need to build up a mental picture of the change, and the larger that change is, the more context needs to be held in focus. Additionally, large PRs tend to develop their own inertia. If a reviewer has a great idea for an alternative approach, but it would require scrapping most of the work already done in the PR, it's not likely to be acted upon (if it's even shared in the first place).
+
+It's easy to say that developers should limit the size of their PRs, but this is very much easier said than done. Github really doesn't make it clear or intuitive how to break work up into multiple reviewable units. PRs are based on branches, and it's not always clear how to juggle multiple branches. Even for folks who are comfortable with Git, the path can be very tricky.
+
+Ideally, the developer could spin up new PRs as they went, allowing them to solicit feedback early, without being blocked while waiting for it. If the feedback _does_ require significant changes, it should be easy to integrate those changes into their more-recent work.
+
+This blog post explains how we solved for these concerns!
+
+# Workflow overview
```

Comment:
This heading should be an h2, since the page title creates an h1. Then, the content hierarchy can follow and be more accessible for users of assistive technology. Here's a link to more info: https://www.gatsbyjs.org/contributing/docs-contributions/#headings

```suggestion
## Workflow overview
```